frontend/views/pages/Home.vue

```diff
@@ -63,12 +63,9 @@ main.c-splash(data-test='homeLogo' v-if='!currentGroupId')
       @click='onInstallClick'
     ) {{ L('Install') }}
 
-  //- TODO: conditionally show this depending on environment variable
-  //- footer.c-footer(v-if='!isLoggedIn')
-  //-   banner-simple.c-demo-warning(severity='warning')
-  //-     i18n(
-  //-       :args='{ a_:`<a class="link" href="https://groupincome.org/beta-testing/" target="_blank">`, _a: "</a>" }'
-  //-     ) This is a beta-testing site. Groups will have to be re-created once Group Income is released. {a_}Read more.{_a}
+  footer.c-footer(v-if='!isLoggedIn && ephemeral.serverMessages.length > 0')
+    banner-simple.c-demo-warning(v-for='text in ephemeral.serverMessages' severity='warning')
```

Comment:
Can you update the styles here to support multiple messages with this `margin-bottom` or `margin-block-end`?

```css
.c-demo-warning {
  text-align: left;
  margin-bottom: 1rem;
}
```